ios/Podfile

```diff
@@ -24,5 +25,28 @@ target 'SampleApp' do
 end
 
 target 'Notification Service' do
-  pod 'CustomerIOMessagingPushAPN', :podspec => 'https://raw.githubusercontent.com/customerio/customerio-ios/1.1.1/CustomerIOMessagingPushAPN.podspec'
+  pod 'CustomerIO/Tracking', '~> 1.1'
+  pod 'CustomerIO/MessagingPushAPN', '~> 1.1'  
 end
+
+
+# Workaround to fix compilation issue with Notification Service trying to compile with 
+# SDK giving errors. See: https://github.com/customerio/customerio-ios/issues/178
+cio_target_names = [
+  'CustomerIOTracking',
+  'CustomerIOMessagingPushAPN',
+  'CustomerIOMessagingPush'
+]
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    if cio_target_names.include? target.name 
+      puts "Modifying target #{target.name}"
+  
+      target.build_configurations.each do |config|
+        puts "Setting build config settings for #{target.name}"
+        config.build_settings['APPLICATION_EXTENSION_API_ONLY'] ||= 'NO'
```

Comment:
This is the right fix for Notification service extension. 👍

Reply:
Well, it's the right temporary workaround 😉

I have a [bug fix branch](https://github.com/customerio/customerio-ios/tree/levi/compile-notification-service) on the iOS SDK that needs some testing (when I get a moment). Then we don't need this anymore.

Reply:
Awesome. If that gets fixed from the SDK then it will be a great advantage.